bin/martinize2

```diff
@@ -972,8 +973,14 @@ def entry():
         itp_paths = []
         # Merge chains if required.
         if args.merge_chains:
-            for chain_set in args.merge_chains:
-                vermouth.MergeChains(chain_set).run_system(system)
+            if "all" not in args.merge_chains:
+                input_chain_sets = [i.split(",") for i in args.merge_chains]
+                for chain_set in input_chain_sets:
+                    vermouth.MergeChains(chains=chain_set, all_chains=False).run_system(system)
+            elif "all" in args.merge_chains and len(args.merge_chains) == 1:
+                vermouth.MergeChains(chains=None, all_chains=True).run_system(system)
```

Comment:
`args.merge_chains` cannot be 'all' and have a length of 1 right? Since the split only happens in the `if`

Reply:
this is defending against multiple arguments though, eg. if you give both `-merge all -merge A,B` then `args.merge_chains = ['all, 'A,B']`, so really you need to make sure there really is only one argument

Reply:
Good catch! Needs a comment though.